product_configurator_sale/wizard/product_configurator.py

```diff
@@ -13,6 +13,24 @@ class ProductConfiguratorSale(models.TransientModel):
     order_id = fields.Many2one(comodel_name="sale.order", required=True, readonly=True)
     order_line_id = fields.Many2one(comodel_name="sale.order.line", readonly=True)
 
+    domain_attr_ids = fields.Many2many(
+        "product.attribute.value",
+        "domain_attrs_values_sale_rel",
+        "wiz_id",
+        "attribute_id",
+        string="Domain",
+    )
+    dyn_field_value = fields.Char()
+
+    domain_attr_2_ids = fields.Many2many(
+        "product.attribute.value",
+        "domain_attrs_2_values_sale_rel",
+        "wiz_id",
+        "attribute_id",
+        string="Domain",
+    )
+    dyn_field_2_value = fields.Char()
+
```

Comment:
@Chirag-OSI I don't see these field being used anywhere. Can you explain what they are for?

Reply:
@dreispt I have removed those unused fields.